pycqed/instrument_drivers/meta_instrument/heterodyne.py

```diff
@@ -95,6 +95,10 @@ def common_init(self, name, LO, AWG, acquisition_instr='CBox',
                            label='Automatic AWG sequence loading',
                            parameter_class=ManualParameter,
                            initial_value=True)
+        self.add_parameter('acq_marker_channels', vals=vals.Strings(),
+                           label='Acquisition trigger channels',
```

Comment:
@antsr , You could use a `ManualParameter` for this. It's located in `qcodes.instrument.parameter`  and negates the use of the set_cmd and get_cmd